### DISCUSSION TITLE
did not find package when i change the default  source

### DISCUSSION THREAD

**Discussion by beincy** [astral-sh/rye#472] [Q&A]
I added configuration to global config .likes
```
[[sources]]
name = "default"
url = "http://172.16.2.13/pypi/simple/"
```

But whenever I run `rye add xxx`, it always gave me the wrong answer. 
`error: did not find package 'xxx'`
It's  works when I use 'pip'.

**Comment by mitsuhiko** [comment]
This one replaces the default source so it will not consult pypi, probably hiding the package `xxx` which is on another index.
